Dockerized environment (#1)
lemonatio/proof-of-liabilities: README.md

- Python 3.9 or above

> Alternatively, if you don’t have python installed, you can use the dockerized version of the script.

Reviewer: ```suggestion
Alternatively, if you don't have python installed, for the `verify.py` script you can use the dockerized version.
```

python3 verify.py -i proof.csv -r root.json
```

> In case python dependencies are not installed a dockerized version of the script with the official python images can be used:

Reviewer: ```suggestion
In case python dependencies are not installed, a dockerized version of the script with the official python images can be used:
```

```bash

> ./veryfy.sh -i proof.csv -r root.json

Reviewer 2: Hay un typo acá. Debería ser `verify` en lugar de `veryfy`

Author: Cambiado, gracias!
